### ISSUE TITLE
.[Adaptive cards-blog]: Search functionality is not provided to search the desired card on the blog page. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8886]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test environment:**  

OS: Win11  

OS Version: 24H2 (OS Build 26100.1)    

URL:[Adaptive Cards](https://adaptivecards.io/) 

Edge dev Browser version: 125.0.2506.0 (Official build) dev (64-bit)  

Screen reader: Narrator. 

**Repro Steps:** 

1. Open URL in the Edge browser & sign in with valid credentials. 
2. It will open the Adaptive cards home page 

3. Activate the blog control and observe the issue. 

**Actual Result:**  

The search functionality is not provided for the users to perform the search efficiently and the users will have to navigate through so many items. 

**Expected Result:** 

The search functionality should be provided for the users to perform the search efficiently, so that the users will not have to navigate through so many items. 

**User Experience:**  

Users will be impacted if the search functionality is not provided so the users will have to perform so many navigations to reach the desired item.

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards/assets/95913748/d973f8a5-ddb5-430f-8165-3592f7ca626f

### Card JSON

```JSON
NA
```
